assets/js/modules/tagmanager/util/validation.js

```diff
@@ -75,6 +75,57 @@ export function isValidContainerID( containerID ) {
 	return typeof containerID === 'string' && /^GTM-[A-Z0-9]+$/.test( containerID );
 }
 
+/**
+ * Gets normalized container name.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} containerName Container name to normalize.
+ * @return {string} Normalized container name.
+ */
+export function getNormalizedContainerName( containerName ) {
```

Comment:
While this is related to validation, I don't think it belongs in this module.

Let's put it in its own module, or a general `utils/container` module.

Please add tests for this function as well. You can probably borrow most of it from the current tests in PHP – see `container_name_provider`.

Reply:
Updated.